multisrc/overrides/mangathemesia/thunderscans/src/ThunderScans.kt

```diff
@@ -0,0 +1,120 @@
+package eu.kanade.tachiyomi.extension.ar.thunderscans
```

Comment:
Put this source in another PR. Only one source per PR.

Reply:
ok i will remove it for now